examples/news-list/client/routes.js

```diff
@@ -0,0 +1,105 @@
+import React from 'react'
+import { Helmet } from 'react-helmet'
+import { connectReadModel } from 'resolve-redux'
```

Comment:
May be separate to particular package?

Reply:
I.e. split components info different files? Some for component, some for containers, some for common code?

Reply:
No, resolve-redux to static and dynamic readmodels. I can not describe good case now, but may be we can think about it

Reply:
Rename `connect` to `connectViewModel` since it appropriate only for view models